### DISCUSSION TITLE
[ERROR] Could not resolve "/app/source.config.ts"

### DISCUSSION THREAD

**Discussion by Manmit124** [fuma-nama/fumadocs#1154] [General]
I am encountering the following error during the Docker build process, even though I have correctly set up source.config.ts

**Comment by Manmit124** [comment]
![image](https://github.com/user-attachments/assets/5e8e78b9-daab-44e6-96a5-8ab7900c32d4)

**Reply by Rahim-444** [reply]
hi,
make sure to COPY the source.config.ts in your dockerfile with : 

COPY source.config.ts ./

before the RUN npm install
